pages/archive/index.js

```diff
@@ -0,0 +1,10 @@
+import FlashCardList from "@/components/FlashCardList";
+
+export default function archive({ onMarkCorrect, flashCards }) {
+  return (
+    <>
+      <h1>Archive</h1>
+      <FlashCardList onMarkCorrect={onMarkCorrect} flashCards={flashCards} />
```

Comment:
nice reuse of your FlashCardList component. Don't forget to pass down here the custom error message.